src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,65 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+
+@Entity
+public class User {
+    @Id
+    @GeneratedValue (strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String email;
+    private String password;

```

Comment:
Let’s add the required properties to these columns. Column(nullable = false/unique = true)